LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2LWAbilitiesModTemplate.uc

```diff
@@ -234,6 +236,14 @@ static function GetUpdatedHitChances(X2AbilityToHitCalc_StandardAim ToHitCalc, o
 	}
 }
 
+static funtion UpdateTeamwork(X2AbilityTemplate Template)
```

Comment:
Not quite sure how this compiled/worked. `funtion` should of course be `function`.